app/paths.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
     },
     credentials: {
       index: '/your-psp/:credentialId/credentials',
-      edit: '/your-psp/:credentialId/credentials/edit'
+      edit: '/your-psp/:credentialId/credentials/edit',
+      checkOrgDetails: '/your-psp/:credentialId/credentials/check-organisation-details'
```

Comment:
Can this be moved to yourPSP->stripeSetup as this is part of stripe onboarding? 

Also the url should be `/your-psp/:credentialId/check-organisation-details`

Reply:
Thank - will update that